server/container_create_linux.go

```diff
@@ -458,13 +458,18 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 		}
 
 		if !isInCRIMounts("/sys", containerConfig.Mounts) {
-			specgen.RemoveMount("/sys")
```

Comment:
What if we are privileged and host network?

Reply:
the mount generating code is all over the place tbh. Right after this block, we overwrite the mount if we're privileged

Reply:
Can we skip the add if we are also privileged since they are added rw in that case?